modules/serverless-installing-cli-linux.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_{context}"]
+= Installing `kn` for Linux
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active OpenShift Enterprise subscription on your Red Hat account:
+
+[source]
+----
+# subscription-manager register
+# subscription-manager refresh
+# subscription-manager attach --pool=<pool_id> <1>
+# subscription-manager repos --enable="openshift-serverless-1-for-rhel-8-x86_64-rpms"
+# yum install atomic-openshift-clients
+----
+<1> Pool ID for an active OpenShift Enterprise subscription
+
+For Linux distributions, you can download the CLI directly from the link:TBD[Red Hat Customer Portal] as a *_tar.gz_* archive. 
+
+
+.Procedure
+
+. Log in with your Red Hat account. You must have an active subscription to access the downloads page.
+
+. Download the link:TBD[CLI] from the Red Hat Customer Portal.
+
+. Unpack the archive. To unpack the archive use the command:
```

Comment:
I think you can leave out "To unpack the archive use the command:". The user should be able to understand this since you're following the step with a command.